src/libraries/System.Numerics.Tensors/Directory.Build.props

```diff
@@ -2,8 +2,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <!-- This is a preview package. Do not mark as stable. -->
-    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
-    <IsShippingAssembly>false</IsShippingAssembly>
+    <IsShipping>false</IsShipping>
```

Comment:
This is wrong. As `System.Numerics.Tensors` is a shipping package but with a non-stable version.

Reply:
Which means it won't make it to nuget.org anymore. I believe we want that to go to nuget.org. I asked this a while ago.
https://github.com/dotnet/runtime/issues/33636#issuecomment-599684187

Reply:
I believe you want to do:

```
<SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

instead

Reply:
Just as we had it. We shouldn't change this package at all. We need `IsShippingAssembly=false` so that the assembly metadata contains the non-stable version.

Reply:
We dont want to ship any rc or rtm versions of these packages. (as we did in 3.1)
We will start shipping the preview 6 versions

Reply:
> <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>

adding this will still ship the rc1 & rc2 version and we dont want that as well.

Reply:
Ok... why instead exclude them from the build list in libraries-packages.proj and don't build them at all?

Reply:
We need to build Microsoft.Extensions.Internal.Transport as it is necessary for aspnetcore.

For System.Numerics.Tensors we just want the smallest change to not produce the shipping package.  As mentioned in the issue we'll remove this code completely in 6.0 and put it back in corefxlab.

Reply:
Hmm, ok.